### ISSUE TITLE
CentOS-5 build error: MS_SLAVE

### ISSUE THREAD

**Issue by truatpasteurdotfr** [apptainer/singularity#163]
reminder for gmk :P 

```
make[2]: Entering directory `/home/tru/git/singularity/src'
if gcc -DHAVE_CONFIG_H -I. -I. -I.  -DSYSCONFDIR=\"/usr/local/etc\" -DLOCALSTATEDIR=\"/usr/local/var\" -DLIBEXECDIR=\"/usr/local/libexec\" -DNS_CLONE_FS -DNS_CLONE_NEWNS -DNO_SETNS -I/c5/shared/git/1.8.2.1/include -Wall -fpie -I/c5/shared/git/1.8.2.1/include -MT sexec-sexec.o -MD -MP -MF ".deps/sexec-sexec.Tpo" -c -o sexec-sexec.o `test -f 'sexec.c' || echo './'`sexec.c; \
        then mv -f ".deps/sexec-sexec.Tpo" ".deps/sexec-sexec.Po"; else rm -f ".deps/sexec-sexec.Tpo"; exit 1; fi
sexec.c: In function ‘main’:
sexec.c:455: error: ‘MS_SLAVE’ undeclared (first use in this function)
sexec.c:455: error: (Each undeclared identifier is reported only once
sexec.c:455: error: for each function it appears in.)
make[2]: *** [sexec-sexec.o] Error 1
make[2]: Leaving directory `/home/tru/git/singularity/src'
make[1]: *** [all] Error 2
make[1]: Leaving directory `/home/tru/git/singularity/src'
make: *** [all-recursive] Error 1
```

**Comment by gmkurtzer** [comment]
I just added a test in master ae06892. Can you test and verify?

Thanks!

**Comment by truatpasteurdotfr** [comment]
it is still broken for me :(

./configure yields:

```
checking for feature: MS_SLAVE... yes
```

and the builds fails at the same place:

```
make[2]: Entering directory `/home/tru/rpm/BUILD/singularity-2.2/src'
if gcc -DHAVE_CONFIG_H -I. -I. -I.  -DSYSCONFDIR=\"/etc\" -DLOCALSTATEDIR=\"/var\" -DLIBEXECDIR=\"/usr/libexec\" -DNS_CLONE_FS -DNS_CLONE_NEWNS -DSINGULARITY_MS_SLAVE -DNO_SETNS -I/c5/shared/git/1.8.2.1/include -Wall -fpie -I/c5/shared/git/1.8.2.1/include -MT sexec-sexec.o -MD -MP -MF ".deps/sexec-sexec.Tpo" -c -o sexec-sexec.o `test -f 'sexec.c' || echo './'`sexec.c; \
        then mv -f ".deps/sexec-sexec.Tpo" ".deps/sexec-sexec.Po"; else rm -f ".deps/sexec-sexec.Tpo"; exit 1; fi
sexec.c: In function 'main':
sexec.c:456: error: 'MS_SLAVE' undeclared (first use in this function)
sexec.c:456: error: (Each undeclared identifier is reported only once
sexec.c:456: error: for each function it appears in.)
make[2]: *** [sexec-sexec.o] Error 1
make[2]: Leaving directory `/home/tru/rpm/BUILD/singularity-2.2/src'
make[1]: *** [all] Error 2
make[1]: Leaving directory `/home/tru/rpm/BUILD/singularity-2.2/src'
make: *** [all-recursive] Error 1
error: Bad exit status from /home/tru/rpm/tmp/rpm-tmp.82369 (%build)
```

**Comment by gmkurtzer** [comment]
ok, I just pushed a fix to master 5e01943 that hopefully will resolve that.

**Comment by truatpasteurdotfr** [comment]
yeah :P fixed
